File: RUTA_REAL_AQUI/backend/app/models/catalog.py
from sqlalchemy import Column, Integer, String, Numeric

from app.database import Base


class ServicePrice(Base):
    """Precios base de lavandería (lavado por kilo, cobertores, tenis, planchado)."""

    __tablename__ = "service_prices"

    id = Column(Integer, primary_key=True, index=True)
    name = Column(String(100), unique=True, nullable=False)
    price = Column(Numeric(10, 2), nullable=False)


class DryCleaningItem(Base):
    """Catálogo completo de tintorería (nombre + precio)."""

    __tablename__ = "dry_cleaning_items"

    id = Column(Integer, primary_key=True, index=True)
    name = Column(String(150), unique=True, nullable=False)
    price = Column(Numeric(10, 2), nullable=False)


class ExpenseCategory(Base):
    """Categorías fijas de gastos (Préstamo, Sueldos, Renta, etc.)."""

    __tablename__ = "expense_categories"

    id = Column(Integer, primary_key=True, index=True)
    name = Column(String(50), unique=True, nullable=False)

File: frontend/src/constants/catalog.js
// Precios base de lavandería (sección 7.B). El catálogo de tintorería
// (7.C) es muy grande y se busca vía API (services/api.js -> searchDryCleaning),
// no se replica aquí para no desincronizarse del backend.
export const SERVICE_PRICES = [
  { name: "Lavado por kilo", price: 35.0 },
  { name: "Cobertor individual", price: 75.0 },
  { name: "Cobertor matrimonial", price: 85.0 },
  { name: "Cobertor queen size", price: 95.0 },
  { name: "Par de tenis", price: 45.0 },
  { name: "Docena de planchado", price: 120.0 },
];

export const EXPENSE_CATEGORIES = [
  "Préstamo",
  "Sueldos",
  "Renta",
  "Jabón",
  "Suavitel",
  "Papel",
  "Bolsas de plástico",
  "Luz",
  "Agua",
  "Otro",
];

export const ORDER_STATUSES = ["Recibido", "Lavadora", "Secadora","Planchado", "Doblado", "Listo"];

export const EMPLOYEES = ["Marian", "Lupita"];
